crates/swc/src/config/mod.rs

```diff
@@ -240,37 +241,106 @@ impl Options {
     }
 }
 
-/// Configuration related to source map generated by swc.
-#[derive(Clone, Serialize, Deserialize, Debug)]
-#[serde(untagged)]
+/// Configuration related to source maps generation.
+#[derive(Debug, Clone, Serialize, Eq, PartialEq)]
 pub enum SourceMapsConfig {
     Bool(bool),
-    Str(String),
+    Inline,
+    Linked(PathBuf),
```

Comment:
I think this should be `Linked`, not `Linked(PathBuf)`

Reply:
No problem. I will revert my previous commit.